G/GeoStatsDevTools/Compat.toml

```diff
@@ -77,7 +77,7 @@ julia = "1"
 ["0.4.4"]
 GeoStatsBase = "0.4.1-*"
 
-["0.4.4-0"]
+["0.4.4-0.4"]
 CSV = "0.2.0-*"
 DataFrames = "0.11.5-*"
 Distances = "0.5.0-*"
```

Comment:
Plase keep these for 0.5

Reply:
What do you mean? I don't know what this Compat.toml file is. The only thing I did was bump the version in Project.toml.

Reply:
I mean, keep the requirements from the last release.